budget/run.py

```diff
@@ -52,6 +52,15 @@ def configure():
         if not 'MAIL_DEFAULT_SENDER' in app.config:
             app.config['MAIL_DEFAULT_SENDER'] = DEFAULT_MAIL_SENDER
 
+    if "pbkdf2:sha256:" not in app.config['ADMIN_PASSWORD'] and app.config['ADMIN_PASSWORD']:
+        # Since v1.1
+        warnings.warn(
```

Comment:
Stop me if I'm wrong, but the message seems misleading: it's not a primary matter of safety reasons: without using an hashed password, **the previously set password won't work anymore**. 
Do you agree me ?